x-pack/plugins/task_manager/README.md

```diff
@@ -49,53 +49,52 @@ The task_manager can be configured via `taskManager` config options (e.g. `taskM
 
 ## Task definitions
 
-Plugins define tasks by adding a `taskDefinitions` property to their `uiExports`.
+Plugins define tasks by calling the `registerTaskDefinitions` method on `server.plugins.taskManager`.
 
 A sample task can be found in the [plugin_functional/sample_task_plugin](../../test/plugin_functional/sample_task_plugin/) folder.
 
 ```js
-{
-  taskDefinitions: {
-    // clusterMonitoring is the task type, and must be unique across the entire system
-    clusterMonitoring: {
-      // Human friendly name, used to represent this task in logs, UI, etc
-      title: 'Human friendly name',
-
-      // Optional, human-friendly, more detailed description
-      description: 'Amazing!!',
-
-      // Optional, how long, in minutes, the system should wait before
-      // a running instance of this task is considered to be timed out.
-      // This defaults to 5 minutes.
-      timeOut: '5m',
-
-      // The clusterMonitoring task occupies 2 workers, so if the system has 10 worker slots,
-      // 5 clusterMonitoring tasks could run concurrently per Kibana instance. This value is
-      // overridden by the `override_num_workers` config value, if specified.
-      numWorkers: 2,
-
-      // The createTaskRunner function / method returns an object that is responsible for
-      // performing the work of the task. context: { taskInstance, kbnServer }, is documented below.
-      createTaskRunner(context) {
-        return {
-          // Perform the work of the task. The return value should fit the TaskResult interface, documented
-          // below. Invalid return values will result in a logged warning.
-          async run() {
-            // Do some work
-            // Conditionally send some alerts
-            // Return some result or other...
-          },
-  
-          // Optional, will be called if a running instance of this task times out, allowing the task
-          // to attempt to clean itself up.
-          async cancel() {
-            // Do whatever is required to cancel this task, such as killing any spawned processes
-          },
-        };
-      }
-    },
+const { client: taskManagerClient } = server.plugins.taskManager;
```

Comment:
Could we flatten the taskManager a bit and just make this:

```js
server.plugins.taskManager.registerTaskDefinitions({ ... })
```

Instead of:

```js
server.plugins.taskManager.taskManagerClient.registerTaskDefinitions({ ... });
```